src/openrct2/Game.h

```diff
@@ -138,6 +138,7 @@ extern uint32_t gCurrentTicks;
 extern uint16_t gTicksSinceLastUpdate;
 extern uint8_t  gGamePaused;
 extern int32_t gGameSpeed;
+enum { GameSpeedPaused = 0, GameSpeedNormal = 2, GameSpeedTurbo = 5, GameSpeedHyper = 6 };
```

Comment:
This shouldn't all be on one line, and moved up with the other enums.

Reply:
Constants and enums should be upper case.